ibm/service/vpc/resource_ibm_is_instance_group_manager_policy.go

```diff
@@ -193,6 +193,12 @@ func resourceIBMISInstanceGroupManagerPolicyUpdate(d *schema.ResourceData, meta
 		updateInstanceGroupManagerPolicyOptions.InstanceGroupID = &instanceGroupID
 		updateInstanceGroupManagerPolicyOptions.InstanceGroupManagerID = &instanceGroupManagerID
 
+		instanceGroupManagerPolicyAsPatch, asPatchErr := instanceGroupManagerPolicyPatchModel.AsPatch()
+		if asPatchErr != nil {
+			return fmt.Errorf("[ERROR] Error calling asPatch for RoutingTablePatchModel: %s", asPatchErr)
```

Comment:
Error calling asPatch for RoutingTablePatch:

Reply:
Change this and upstream.

Reply:
ok